man/systemd.link.xml

```diff
@@ -47,9 +47,9 @@
 
     <para>Along with the link file <filename>foo.link</filename>, a "drop-in" directory
     <filename>foo.link.d/</filename> may exist. All files with the suffix <literal>.conf</literal>
-    from this directory will be parsed after the file itself is parsed. This is useful to alter or add
-    configuration settings, without having to modify the main configuration file. Each drop-in file
-    must have appropriate section headers.</para>
+    from this directory will be merged in the alphanumeric order and parsed after the file itself is
```

Comment:
I think this needs to say "after the *main* file itself *has been* parsed". Otherwise it's not clear what "file" refers to.